Source/RapyutaSimulationPlugins/Private/Tools/SimulationState.cpp

```diff
@@ -256,22 +256,35 @@ void ASimulationState::SpawnEntitySrv(UROS2GenericSrv* Service)
 
             FActorSpawnParameters SpawnParameters;
             SpawnParameters.Name = FName(Request.state_name);
+
             FRotator Rotator = Request.state_pose_orientation.Rotator();
             FVector Position(Request.state_pose_position_x, Request.state_pose_position_y, Request.state_pose_position_z);
-            AActor* NewEntity = GetWorld()->SpawnActor(SpawnableEntities[Request.xml], &Position, &Rotator, SpawnParameters);
+            FVector Scale(1, 1, 1);
+            FTransform Transform(Rotator, Position, Scale);

```

Comment:
We could use Scale as `FVector::OneVector` but I suppose its value is defaulted in `FTransform` ctor.

Reply:
ohh Ill try that instead!